interfaces/builtin/docker_support.go

```diff
@@ -55,6 +55,8 @@ const dockerSupportConnectedPlugAppArmorCore = `
 # /system-data/var/snap/docker/common/var-lib-docker/overlay2/$SHA/diff/
 /system-data/var/snap/{@{SNAP_NAME},@{SNAP_INSTANCE_NAME}}/common/{,**} rwl,
 /system-data/var/snap/{@{SNAP_NAME},@{SNAP_INSTANCE_NAME}}/@{SNAP_REVISION}/{,**} rwl,
+# allow unrestricted use of posix message queues
```

Comment:
The failing test is actually a case in classic, so we need this feature enabled for `dockerSupportConnectedPlugAppArmor` as well, not only for core. 

I propose a strategy similar to `userns`, where add specific `mqueue` snippet if the feature is probed and proven to be available.

Reply:
Ah apologies, I rushed this.